src/protagonist.h

```diff
@@ -48,7 +48,7 @@ namespace protagonist {
 
         // Wraps snowcrash::Warnings and snowcrash:Blueprint into report object
         // Note: snowcrash::Result::Error is being sent separately as Error object
-        static v8::Local<v8::Object> WrapResult(const snowcrash::ParseResult<snowcrash::Blueprint>& parseResult,
+        static v8::Local<v8::Object> WrapResult(snowcrash::ParseResult<snowcrash::Blueprint>& parseResult,
```

Comment:
If that is the case and you need to mutate parserResult shouldn't the function signature follow the rest of Snow Crash and have mutable parameters at the end (after those marked `const`)?